clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,153 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 5}}
+{{$MEDIUM_BACKEND_SIZE := 3}}
+{{$SMALL_BACKEND_SIZE := 1}}
+{{$LARGE_BACKEND_LABEL := "ilblarge"}}
```

Comment:
nit:

"ilb-large" "ilb-medium", "ilb-small" would be more readable

Reply:
done